### ISSUE TITLE
7 - Pre-fill form with data in case there is an existing climbed_boulder entry for this user and boulder

### ISSUE THREAD

**Issue by PulexIrritans** [PulexIrritans/Heelhook-Capstone#11]
## Value statement - a requirement, not a solution
As a climber when I enter the "add" page for a specific boulder
I want to see the data in the form that I have entered earlier  
so that I can see my latest entry.

## Description (Text, Scribble, Wireframe, Design)
Here is an example of how the object should look like that the server sends on fetch:
{ _id: "999999", climber_id: "9999999999", boulder_id: "3", date: Date Mon May 02 2022 10:16:38 GMT+0200 (Central European Summer Time), projected: false, attempts: "4", result: "top", liked: true, level_feedback: 0 }

## Acceptance criteria
- [ ] The form is prefilled with the most recent "climbed_boulder" data for the selected boulder when entering the "add" page.
- [ ] If for the selected boulder no climb data is existing then the form will be empty. 

## Tasks
- [ ] add a GET route for a specific boulder id to check for existing entries when the add page with the form is accessed.
- [ ] implement logic to the server to look for a "climbed_boulder" which contains the specific user id (hard-coded for now) and boulder id.
- [ ] implement error message in case no entry could be found
- [ ] in the frontend fetch the climbed_boulder data and prefill the form with the response data or keep the form empty if no data

## Size
(medium)

**Comment by PReichetanz** [comment]
I would rephrase the value proposition:
As a climber
I want to see the addForm prefilled with data
so that I can see my latest entry.

I don't understand your task regarding the error message: Where should it be displayed? You want the form to be empty if nothing can be found, so why do you consider this as an error?

**Comment by PulexIrritans** [comment]
@PReichetanz 
Thank you! I have updated the story accordingly.
